examples/one_dot_one/src/index.ts

```diff
@@ -87,6 +87,19 @@ export function go(basePath: string) {
           }],
           status: "ACTIVE",
         },
+        {
+          apiName: "create-todo",
```

Comment:
Action API Names are very unfortunately kebab-case :( Gateway is smart enough to handle this, but will eventually need a long term solution to migrate API Names